### DISCUSSION TITLE
Windows Explorer snap to desktop edges

### DISCUSSION THREAD

**Discussion by Jamesstevens-3rd** [valinet/ExplorerPatcher#1764] [Ideas]
There is a feature to Snap to Desktop Edges in Media Player Classic BE that I find very useful and would welcome something similar in Explorer Patcher, should work well with Multi-Tasking Snap Windows

**Comment by pyrates999** [comment]
You mean in the explorer patcher properties application?

**Reply by Jamesstevens-3rd** [reply]
Yeah

the way MPC-BE implements this is when dragging the window towards an edge of the desktop its like a magnet pulling it from about 10 pixels away

**Comment by pyrates999** [comment]
I don't think the current design of it allows for that.  But perhaps someone will implement it.  Explorer patcher is open source, so you could try to implement it if you wanted to.
